Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -2,21 +2,25 @@
 
 ''' IMPORTS '''
 from collections import defaultdict
-from typing import Any, Dict
+from typing import Any, Dict, Tuple, List
 import httplib2
 import urllib.parse
 from oauth2client import service_account
+from copy import deepcopy
+import dateparser
+from hashlib import sha256
 
 # A request will be tried 3 times if it fails at the socket/connection level
 httplib2.RETRIES = 3
 
 ''' CONSTANTS '''
 
-DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Why was this change necessary?

Reply:
@guykeller With respect to some of our observations, we found out that the timestamps on Chronicle could be more precise to a fraction of seconds, so we modified the `DATE_FORMAT` here.
Moreover, we have also modified the logic for parsing the date related arguments in the command according to newer XSOAR conventions and have used `dateparser` library rather than strictly asking the user to enter the data in a specific format.

Reply:
> Why was this change necessary?

@guykeller With respect to some of our observations, we found out that the timestamps on Chronicle could be more precise to a fraction of seconds, so we modified the DATE_FORMAT here.
Moreover, we have also modified the logic for parsing the date related arguments in the command according to newer XSOAR conventions and have used dateparser library rather than strictly asking the user to enter the date in a specific format.